### PR TITLE
Use Makie for plots in Docs/Stochastic Forcing + new Docs/Vizualize output section + fix some docstrings

### PR DESCRIPTION
**Pull request by navidcy**

### CODE REVIEW

**Comment by jbisits** [comment]
Hope it was ok to push this small change to this PR!

File: docs/src/index.md [hunk 1]
```diff
@@ -50,9 +50,8 @@ Examples aim to demonstrate the main functionalities of each module. Have a look
 
 ## Developers
 
-The development of GeophysicalFlows.jl started by [Navid C. Constantinou](http://www.navidconstantinou.com) and [Gregory L. Wagner](https://glwagner.github.io) during the 21st AOFD Meeting 2017. During the 
-course of time various people have contributed to GeophysicalFlows.jl, including 
-[Lia Siegelman](https://scholar.google.com/citations?user=BQJtj6sAAAAJ), [Brodie Pearson](https://brodiepearson.github.io), and [André Palóczy](https://scholar.google.com/citations?user=o4tYEH8AAAAJ) (see the [example in FourierFlows.jl](https://fourierflows.github.io/FourierFlowsDocumentation/stable/literated/OneDShallowWaterGeostrophicAdjustment/)).
+The development of GeophysicalFlows.jl started by [Navid C. Constantinou](http://www.navidconstantinou.com) and [Gregory L. Wagner](https://glwagner.github.io) during the 21st AOFD Meeting 2017. Since then various people have contributed to GeophysicalFlows.jl, including
```
Comment (jbisits) [inline docs/src/index.md:53]: "The development of GeophysicalFlows.jl **was** started by..."?

File: docs/src/modules/barotropicqgql.md [hunk 2]
```diff
@@ -83,8 +83,9 @@ and [`addforcing!`](@ref GeophysicalFlows.BarotropicQGQL.addforcing!).
 All required parameters are included inside [`Params`](@ref GeophysicalFlows.BarotropicQGQL.Params)
 and all module variables are included inside [`Vars`](@ref GeophysicalFlows.BarotropicQGQL.Vars).
 
-For decaying case (no forcing, ``F = 0``), `vars` can be constructed with [`DecayingVars`](@ref GeophysicalFlows.BarotropicQGQL.DecayingVars). 
-For the forced case (``F \ne 0``) the `vars` struct is with [`ForcedVars`](@ref GeophysicalFlows.BarotropicQGQL.ForcedVars) or [`StochasticForcedVars`](@ref GeophysicalFlows.BarotropicQGQL.StochasticForcedVars).
+For decaying case (no forcing, ``F = 0``), variables are constructed with [`Vars`](@ref GeophysicalFlows.BarotropicQGQL.Vars).
```
Comment (jbisits) [inline docs/src/modules/barotropicqgql.md:86]: "For the decaying case..."?

File: docs/src/modules/singlelayerqg.md [hunk 3]
```diff
@@ -68,8 +68,9 @@ and [`addforcing!`](@ref GeophysicalFlows.SingleLayerQG.addforcing!).
 All required parameters are included inside [`Params`](@ref GeophysicalFlows.SingleLayerQG.Params)
 and all module variables are included inside [`Vars`](@ref GeophysicalFlows.SingleLayerQG.Vars).
 
-For decaying case (no forcing, ``F=0``), `vars` can be constructed with [`DecayingVars`](@ref GeophysicalFlows.SingleLayerQG.DecayingVars). 
-For the forced case (``F \ne 0``) the `vars` struct is with [`ForcedVars`](@ref GeophysicalFlows.SingleLayerQG.ForcedVars) or [`StochasticForcedVars`](@ref GeophysicalFlows.SingleLayerQG.StochasticForcedVars).
+For decaying case (no forcing, ``F = 0``), variables are constructed with [`Vars`](@ref GeophysicalFlows.SingleLayerQG.Vars).
```
Comment (jbisits) [inline docs/src/modules/singlelayerqg.md:71]: "For the decaying case..."? This looks to be the same where it appears in subsequent scripts so if you prefer it as is totally fine.

File: docs/src/visualize.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Visualize output
+
+In the examples we use [Makie.jl](https://makie.juliaplots.org/stable/) for plotting.
+
+Makie comes with few [backends](https://makie.juliaplots.org/stable/#makie_ecosystem). In the documented examples
```
Comment (jbisits) [inline docs/src/visualize.md:5]: "Makie comes with a few..."?

File: docs/src/visualize.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+we use [CairoMakie](https://makie.juliaplots.org/stable/documentation/backends/cairomakie/) since this backend
+works well on headless devices, that is, devices without monitor. Since the documentation is automatically
+build via GitHub actions the CairoMakie backend is necessary. Users that run GeophysicalFlows.jl on
```
Comment (jbisits) [inline docs/src/visualize.md:8]: build -> built

GeophysicalFlows.jl -> `GeophysicalFlows.jl` maybe even with the link?

File: docs/src/visualize.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+devices with a monitor they might want to change to [GLMakie](https://makie.juliaplots.org/stable/documentation/backends/glmakie/)
```
Comment (jbisits) [inline docs/src/visualize.md:9]: Can remove the "they", "devices with a monitor might want..."

File: docs/src/visualize.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+that display figures in an interactive window.
```
Comment (jbisits) [inline docs/src/visualize.md:10]: which displays figures in an interactive window.

File: docs/src/visualize.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+In GeophysicalFlows.jl simulations, we can either visualize the flow on-the-fly as the problem is stepped forward or
```
Comment (jbisits) [inline docs/src/visualize.md:12]: GeophsyicalFlows.jl -> `GeophsyicalFlows.jl` again maybe with the link?

**Review by jbisits — CHANGES_REQUESTED** [review]
Just some phrasing things.

File: docs/src/visualize.md (hunk 7 above)
Comment (navidcy) [reply docs/src/visualize.md:10]: are you sure it's "which" here? I thought it should be "that"... but I'm not a native English speaker...

**Comment by navidcy** [comment]
thanks @jbisits, I did most of them!!

File: docs/src/visualize.md (hunk 7 above)
Comment (jbisits) [reply docs/src/visualize.md:10]: Yes now I look at it again my suggestions is perhaps not that good. Maybe clearest is "...change to the GLMakie backend that displays figures in an interactive window.

Comment (jbisits) [reply docs/src/visualize.md:10]: Or maybe just display -> displays? Likely have overthought it!!

**Review by jbisits — CHANGES_REQUESTED** [review]

**Review by jbisits — APPROVED** [review]

**Pull request merged**